acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -68,7 +72,8 @@ def run_pcp_broker(host, instance=0)
 
   lein_command = "cd #{GIT_CLONE_FOLDER}/pcp-broker#{instance}; export LEIN_ROOT=ok; \
      lein with-profile #{LEIN_PROFILE} tk </dev/null >/var/log/puppetlabs/pcp-broker.log.#{SecureRandom.uuid} 2>&1 &"
-  lein_command = append_jvm_limits(lein_command) if host[:gke_container]
+  lein_command = append_jvm_opts('-Xms2g -Xmx2g', lein_command) if host[:gke_container]
```

Comment:
Is this method used anywhere else? Breaking this in to a method that does some complicated string manipulation makes this pretty hard to reason about. It looks like the only special case is if the host is a GKE cluster right? So could we just do something simple like:

```ruby
jvm_opts = ['-Djdk.tls.ephemeralDHKeySize=2048']
jvm_opts << '-Xms2g -Xmx2g' if host[:gke_container]
lein_command = "export JVM_OPTS='#{jvm_opts.join(' ')}'; export LEIN_JVM_OPTS='#{jvm_opts.join(' ')}'; #{lein_command}"
'''
```